.travis.yml

```diff
@@ -48,6 +48,7 @@ env:
     - TRAVIS_FLAVOR=haproxy FLAVOR_VERSION=1.6.9
     - TRAVIS_FLAVOR=haproxy FLAVOR_VERSION=1.5.11
     - TRAVIS_FLAVOR=haproxy FLAVOR_VERSION=1.4.26
+    - TRAVIS_FLAVOR=process FLAVOR_VERSION=latest
```

Comment:
This doesn't require infra so can we make this a default check instead of having its own place on the matrix?